### PR TITLE
Fixed french export email

### PR DESCRIPTION
**Pull request by Cali0707**

### CODE REVIEW

**Comment by Cali0707** [comment]
This resolves https://lite-farm.atlassian.net/browse/F21-650

File: packages/api/src/templates/locales/fr.json [hunk 1]
```diff
@@ -90,12 +90,12 @@
...
-    "SUBJECT": "EXPORT.SUBJET",
-    "PRIVACY": "EXPORT.CONFIDENTIALITÉ",
-    "LANGUAGE_DELAY": "EXPORT.DÉLAIS.LANGUE"
+    "HI": "Bonjour",
+    "SEASON": "Nous espérons que vous avez eu une saison fantastique à",
+    "CERTIFICATION": " Vous pouvez cliquer sur le lien ci-dessous pour télécharger un fichier zip de vos documents de certification et tous les documents à l'appui (photos d'emballages de semences, reçus d'entrée, etc.) pour votre période de déclaration.",
+    "GET_YOUR_EXPORT": "Obtenir vos documents",
+    "SUBJECT": "Export de documents de certification de LiteFarm",
+    "PRIVACY": "Chez LiteFarm, nous prenons votre vie privée au sérieux, donc vous devrez être connecté avant que vous puissiez accéder votre export. Désolée pour le dérangement. Bonne chance avec votre soumission et merci d'utiliser LiteFarm!",
+    "LANGUAGE_DELAY": ""
```
Comment (smattingly) [inline packages/api/src/templates/locales/fr.json:99]: Is this empty string value intentional?

**Review by smattingly — COMMENTED** [review]
See question about empty string.

File: packages/api/src/templates/locales/fr.json (hunk 1 above)
Comment (Cali0707) [reply packages/api/src/templates/locales/fr.json]: Good catch! I totally forgot to translate that one. It should be fixed now.

**Review by smattingly — APPROVED** [review]

**Pull request merged**